specs/signature-specification.md

```diff
@@ -125,6 +73,31 @@ The Notary Project signature payload is a JSON document with media type `applica
   }
 }
 ```
+##### Blob Payload
+
+```jsonc
+{
+  "targetArtifact": {
+    "mediaType": "application/octet-stream",
+    "digest": "sha256:2f3a23b6373afb134ddcd864be8e037e34a662d090d33ee849471ff73c873345",
+    "size": 1024
+  }
+}
+```
+##### Blob Payload with annotations
+
+```jsonc
+{
+  "targetArtifact": {
+    "mediaType": "application/octet-stream",
+    "digest": "sha256:2f3a23b6373afb134ddcd864be8e037e34a662d090d33ee849471ff73c873345",
+    "size": 1024,
+    "annotations": {
+        "io.wabbit-networks.buildId": "123"  // user defined metadata
+    }
+  }
+}
+```
```

Comment:
Since this are similar to OCI, IMO we should remove them and call them just `payload`.